### ISSUE TITLE
Question on the Amount of Labeled Data Used to Train the ResNet Feature Extractor

### ISSUE THREAD

**Issue by cpphoo** [Yikai-Wang/ICI-FSL#6]
Hi, thanks for your excellent work! 

After reading your paper, I was wondering whether the ResNet backbone was trained on the labelled portion of the base dataset (i.e. 10% of examples in each base class for tieredImageNet or 40% of examples in each base class in MiniImageNet as reported in [Ren et. al.](https://arxiv.org/abs/1803.00676)) or 100% of the images from each base class in the dataset.

**Comment by Yikai-Wang** [comment]
We use 90% of examples in each base class in all experiments for training set, and the residual 10% are for validation set.
Details can be found in
https://github.com/Yikai-Wang/ICI-FSL/blob/f2537b7742159adfa68cc8646b9a20c570320443/datasets.py#L151-L161

**Comment by cpphoo** [comment]
Got it! Thanks for your prompt reply!
